app/src/main/AndroidManifest.xml

```diff
@@ -14,11 +18,8 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.Rankine">
 
-        <activity
-            android:name=".ui.createRank.CreateRankSet"
+        <activity android:name=".ui.main.MainActivity"
             android:label="Rankine">
-        </activity>
-        <activity android:name=".ui.main.MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <action android:name="android.intent.action.SEARCH" />
```

Comment:
Edits to this file consist of changing permissions and deleting an old activity in the manifest